src/helm/benchmark/runner.py

```diff
@@ -125,6 +134,11 @@ def run_one(self, run_spec: RunSpec):
         scenario.output_path = os.path.join(self.scenarios_path, scenario.name)
         ensure_directory_exists(scenario.output_path)
 
+        # This 'output_path' will be used when the model's input instances are saved.
+        args_str = ",".join([f"{k}={v}" for k, v in sorted(run_spec.scenario_spec.args.items())])
+        input_instances_output_path = os.path.join(self.instances_path, f"{scenario.name}:{args_str}")
```

Comment:
Should we got with no colon when args_str is empty? i.e. just `boolq` rather than `boolq:`?

Reply:
Done.